pages/linux/ac.md

```diff
@@ -17,3 +17,6 @@
 - Print how long a particular user has been connected in hours per day (with total):
 
 `ac --daily-totals --individual-totals {{username}}`
+
+- Short for --reboot --supplants --timewraps
```

Comment:
What  does this actually do though? The description should give a concise explanation of what the command actually does, rather than mention that it is short for something that isn't mentioned elsewhere in the page.

Reply:
I see. If it's just additional information on top of the previous command, we could do this:
```suggestion
- Also display additional details:
```